libs/ts-rest/core/src/lib/client.spec.ts

```diff
@@ -362,7 +383,10 @@ describe('client', () => {
 
       const result = await client.posts.getPosts({});
 
-      expect(result).toStrictEqual({ body: 'string', status: 200 });
+      expect(result.body).toStrictEqual('string');
+      expect(result.status).toBe(200);
+      expect(result.headers.get('Content-Length')).toBe('6');
+      expect(result.headers.get('Content-Type')).toBe('text/plain ');
```

Comment:
Tests failing because of this

```suggestion
      expect(result.headers.get('Content-Type')).toBe('text/plain');
```